deps/Makefile

```diff
@@ -670,6 +670,7 @@ $(eval $(call LLVM_PATCH,llvm-3.7.1))
 else ifeq ($(LLVM_VER),3.7.1)
 $(eval $(call LLVM_PATCH,llvm-3.7.1))
 $(eval $(call LLVM_PATCH,llvm-3.7.1_2))
+$(eval $(call LLVM_PATCH,llvm-3.7.1_3))
 $(LLVM_SRC_DIR)/llvm-3.7.1_2.patch-applied: $(LLVM_SRC_DIR)/llvm-3.7.1.patch-applied
```

Comment:
If I understand the code correctly, this ensures the patches are applied in the correct order. This doesn't make any difference if they don't touch the same files, so adding another line may not be needed here.